### PR TITLE
Add vocabs.

### PR DESCRIPTION
**Pull request by mlhale7**
**JIRA Issue**: [DIGITAL-1461](https://jirautk.atlassian.net/browse/DIGITAL-1461)

## What does this Pull Request do? Any issues?

This PR adds all vocabs to the vocabs.yml file. Note that a few vocabs do not technically include a "dereferenceable_ontology" in that field (wikidata, fast, creativecommons). API links are used for wikidata and creativecommons. I ended up included simply the full RDF file for FAST - https://researchworks.oclc.org/researchdata/fast/FASTAll.nt.zip I'm unsure if this is correct, but there's more documentation on the API here - https://www.oclc.org/developer/api/oclc-apis/fast-api/assign-fast.en.html 

## What's new?

* Adds the following vocabs with values for "dereferenceable_ontology" and "label_property":
* wikidata
* ulan
* aat
* lcgft
* fast
* geonames
* lcsh
* tgm
* iso639-2b
* creativecommons

## How should this be tested?

Make sure that the selected "ontology links" will work with Hyku.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+.DS_Store
```
Comment (markpbaggett) [inline .gitignore]: Remove this or just `git rm .gitignore` for now.  I don't like this sort of thing in the repo ignore.  This should be in your global .gitignore (not the repository .gitignore).

Comment (markpbaggett) [reply .gitignore]: You'll need to google how to set this up on mac because it may not be standard.

File: extras/vocabs.yml [hunk 2]
```diff
@@ -5,3 +5,34 @@ vocabs:
...
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  tgm:
+    dereferenceable_ontology: http://id.loc.gov/vocabulary/graphicMaterials
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  iso639-2b:
+    dereferenceable_ontology: http://id.loc.gov/vocabulary/iso639-2
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  creativecommons:
+    dereferenceable_ontology: https://api.creativecommons.org/rest/1.5/details?license-uri=
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
```
Comment (markpbaggett) [inline extras/vocabs.yml]: Where are you getting this from?  Shouldn't it be `http://creativecommons.org/ns#license-name`?

```suggestion
    label_property: http://creativecommons.org/ns#license-name
```

**Review by markpbaggett — CHANGES_REQUESTED** [review]
Remove the ignore file or at least the .DS_Store line. I don't see skos in creativecommons.  I didn't check every single ontology, but the ones I did looked good except that one.

File: extras/vocabs.yml (hunk 2 above)
Comment (mlhale7) [reply extras/vocabs.yml]: @markpbaggett - this is addressed, thanks.

**Review by markpbaggett — APPROVED** [review]
:shipit:

**Pull request merged**